spec/grocery_list_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../lib/grocery_list'
+
+describe GroceryList do
+subject(:grocery_list) { GroceryList.new(items: ["beer,..."])}
```

Comment:
Can indent one here. Also, put real items in. The `...` is not useful. What we're trying to do is hardcode some values in and test what comes out.